Assets/Scripts/API/BlocksFile.cs

```diff
@@ -584,8 +584,7 @@ private void ReadRmbFldHeader(BinaryReader reader, int block)
                 blocks[block].DFBlock.RmbBlock.FldHeader.BuildingDataList[i].Unknown3 = reader.ReadUInt32();
                 blocks[block].DFBlock.RmbBlock.FldHeader.BuildingDataList[i].Unknown4 = reader.ReadUInt32();
                 blocks[block].DFBlock.RmbBlock.FldHeader.BuildingDataList[i].FactionId = reader.ReadUInt16();
-                blocks[block].DFBlock.RmbBlock.FldHeader.BuildingDataList[i].Sector = reader.ReadInt16();
```

Comment:
Just to expand - I can see they belong together, but I like having them separated this way. The "sector" value, or those 16 bits of the ID are highly unknown at this time. I had to work around this while building the quest system, and now I'd rather not pollute the known good locationID value with unknown data. I'd only have to shift out to the parts I needed anyway. I'm happy for this to remain separated for legacy purposes. :)